opentelemetry/TrackException/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" packages="io.opentelemetry.instrumentation.log4j.appender.v2_17">
+    <Appenders>
+        <Console name="ConsoleAppender" target="SYSTEM_OUT" follow="true">
+            <PatternLayout pattern="log4j2: %d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+        <OpenTelemetry name="OpenTelemetryAppender" captureMapMessageAttributes="true" captureExperimentalAttributes="true"/>
+    </Appenders>
+    <Loggers>
+        <Root level="DEBUG">
```

Comment:
I realized, a more practical configuration (for real apps) would be to have Root at `INFO`, and then override the level for `com.azure.monitor.opentelemetry.exporter` to `DEBUG`. I think this would also more clearly express what we are trying to do with this configuration (which is to enable az mon exporter debug logging)